test/thetest.proto

```diff
@@ -582,3 +584,18 @@ message UnrecognizedWithEmbed {
 	optional Embedded embedded = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false];
 	optional string Field2 = 2;
 }
+
+message ValidatorMessage {
```

Comment:
Lets put this ValidatorMessage in a separate folder in test, lets say test/valdator/...